app/api/v2/__init__.py

```diff
@@ -0,0 +1,29 @@
+from flask import Blueprint
+from flask_restful import Api
+
+# Defines the blueprint application
```

Comment:
Kindly ensure your have a clean code, avoid presenting unnecessary commented out codes.

Reply:
Hi,
I have noted this worthy feedback and will comply with it to remove all blocked out code.

Thank you.

Regards
Steven Ennis